home-application/src/main/java/com/softserveinc/ita/homeproject/application/config/rsql/RSQLConfig.java

```diff
@@ -23,10 +22,11 @@ public void init() {
         rsqlEndpointConfigSet.forEach(config -> {
             Class<? extends BaseEntity> entityClass = config.getQueryConfig().getEntityClass();
             for (Map.Entry<? extends QueryParamEnum, String> entry : config.getMappings().entrySet()) {
-                RSQLJPASupport.addMapping(entityClass, entry.getKey().getParameter(), entry.getValue());
+                io.github.perplexhub.rsql.RSQLCommonSupport
```

Comment:
Can we use import instead of qualified import?

Reply:
Moved to import section.